### PR TITLE
Increases the Size of Questions of Practice mode

### PR DESCRIPTION
**Pull request by SAGARSINGH-1**
Fixes : #21 

here is the issue : https://github.com/eduardconstantin/azure-fundamentals/issues/21

- [X] I have read and agree to the [Code of Conduct](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: pages/api/og.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function handler(req: NextRequest) {
     return new ImageResponse(
       (
         <div tw="flex text-center w-full h-full bg-slate-800 items-center justify-center">
-          <p tw={`px-8 ${width < 640 ? "text-3xl" : "text-xl"} font-sans text-stone-300`}>{question}</p>
+          <p tw={`px-8 ${width < 640 ? "text-3xl" : "text-3xl"} font-sans text-stone-300`}>{question}</p>
```
Comment (eduardconstantin) [inline pages/api/og.tsx:17]:
```suggestion
          <p tw={`px-8 ${width < 640 ? "text-3xl" : "text-2xl"} font-sans text-stone-300`}>{question}</p>
```

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

File: pages/api/og.tsx (hunk 1 above)
Comment (SAGARSINGH-1) [reply pages/api/og.tsx]: Changed to "2xl"

**Comment by eduardconstantin** [comment]
@SAGARSINGH-1 for the next PR make sure you create a branch from dev branch, not main.

**Comment by eduardconstantin** [comment]
@SAGARSINGH-1 I can't merge from dev on your branch, you need to do it

**Review by SAGARSINGH-1 — COMMENTED** [review]
All changes have been made.

**Comment by eduardconstantin** [comment]
Perfect! Thank you!

File: pages/api/og.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default function handler(req: NextRequest) {
     return new ImageResponse(
       (
         <div tw="flex text-center w-full h-full bg-slate-800 items-center justify-center">
-          <p tw={`px-8 ${width < 640 ? "text-3xl" : "text-xl"} font-sans text-stone-300`}>{question}</p>
+          <p tw={`px-8 ${width < 640 ? "text-2xl" : "text-2xl"} font-sans text-stone-300`}>{question}</p>
```
Comment (eduardconstantin) [inline pages/api/og.tsx:17]: let's keep this 3xl
```suggestion
          <p tw={`px-8 ${width < 640 ? "text-3xl" : "text-2xl"} font-sans text-stone-300`}>{question}</p>
```

Comment (SAGARSINGH-1) [reply pages/api/og.tsx:17]: ok

**Review by SAGARSINGH-1 — COMMENTED** [review]
Updated to "3xl"

**Review by eduardconstantin — APPROVED** [review]

**Pull request merged**